cogs/ColourRole.py

```diff
@@ -276,10 +276,10 @@ def calculate_custom_colour_role_position(self, guild: discord.Guild) -> int:
             return 1
         if protected_role_list is None or len(protected_role_list) == 0:
             bot_member: discord.Member = guild.get_member(self.bot.user.id)
-            role_pos = sorted(bot_member.roles, key=lambda x: x.position)[0].position - 1
+            role_pos = sorted(bot_member.roles, key=lambda x: x.position)[-1].position - 1
         else:
             sorted_protected_role_list: List[discord.Role] = sorted(protected_role_list, key=lambda x: x.position)
-            role_pos = sorted_protected_role_list[0].position - 1
+            role_pos = sorted_protected_role_list[0].position
```

Comment:
Were these issues before? or was this accidental. If it was a previous issue please lmk, and add it to CHANGELOG

Reply:
This should've been that way since before ColourRole was merged with master. Not sure when/why it's changed, but it was pre- anything to do with merging into master.

Reply:
Worth putting something like 'minor fixes' in the changelog so if for some strange reason people say it's breaking we at least know something for colour role was changed in this update

Reply:
k will do.